setup.cfg

```diff
@@ -27,8 +27,8 @@ classifiers =
 [options]
 packages = find:
 install_requires =
-    setuptools>=45
-    tomli>=1.0
+    packaging>=20.0
+    setuptools
```

Comment:
Do you know what the actual minimum is? 31? Might be nice here even just for documentation?

Reply:
the tested minimums are part of the long error now

Reply:
Yes, but this is a good place to easily see the minimums, rather than after the error has been triggered. And it's not as "public" as the minimums listed in the docs, which could be 45+.